src/views/HomeView.vue

```diff
@@ -264,11 +275,12 @@ p {
 
 #game-list {
   box-sizing: border-box;
-  background: #cdd1d4;
+  background: #FFF4D7;
```

Comment:
It's best to use the theme color from Vuetify
```suggestion
  background: rgba(var(--v-theme-surface-2));
```